build_tools/third_party/tensorflow/tensorflow/compiler/mlir/xla/CMakeLists.txt

```diff
@@ -48,7 +48,7 @@ external_cc_library(
     "transforms/lower_general_dot.cc"
     "transforms/materialize_broadcasts.cc"
     "transforms/unfuse_batch_norm.cc"
-    "transforms/xla_hlo_fusion.cc"
+    "transforms/mhlo_fusion.cc"
```

Comment:
@joker-eph Is the assumption correct that renaming `xla_hlo_fusiuon.cc` to `mhlo_fusion.cc` was not yet released to the public? Just asking, because I have https://github.com/google/iree/pull/2400 pending, which fixes the CMake build with regard to the changes you introduced with tensorflow/tensorflow@619cc68.

Reply:
Right. It seems like @GMNGeoffrey set it up to open PR here before the code lands internally.

Reply:
Thanks for clarifying :)

Reply:
Yup that's correct. We use the generated PR to run OSS tests and for some additional visibility. It doesn't really work when there are cross-project changes internally though. I should add some documentation so people know what's going on with these